runtime/onert/api/include/nnfw_experimental.h

```diff
@@ -220,6 +220,15 @@ typedef struct nnfw_train_info
   NNFW_TRAIN_OPTIMIZER opt = NNFW_TRAIN_OPTIMIZER_SGD;
 } nnfw_train_info;
 
+/**
+ * @brief Set train_info into session
+ * @note  This function should be called after calling {@link nnfw_load_model_from_file}
+ *        and before calling {@link nnfw_prepare}
+ *
+ *  @return @c NNFW_STATUS_NO_ERROR If successful
+ */
+NNFW_STATUS nnfw_train_set_traininfo(nnfw_session *session, const nnfw_train_info *train_info);
```

Comment:
**This API should be called after model_loaded.**

Because.. If the user set train_info before the model loaded,
train_info in the session can be overwritten while the model loaded.
This can be confusing.

So, To avoid unexpected overwritten while the model loaded, Let's make sure that nnfw_train_set_traininfo is called after the model loaded.

Reply:
Please remove 'into session" in description as I suggested. I think "into session" is better to be removed.

Reply:
updated :)